ChangeLog.md

```diff
@@ -117,6 +131,7 @@
 * Decreased minimal density of obstacles on undergound level of the map
 * Density of objects should now closely resemble H3 RMG
 * Generator will now avoid routing road under guarded objects whenever possible
+* Generator will now avoid placing guards near roads
```

Comment:
Generator will not place a guard near the road if it's stronger than 1/3 of max guard strength for this zone.